<ui version="4.0" >
 <class>RecorderClass</class>
 <widget class="QMainWindow" name="RecorderClass" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>415</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Recorder</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="images.qrc" >
    <normaloff>:/icons/images/record.png</normaloff>:/icons/images/record.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <layout class="QVBoxLayout" name="verticalLayout" >
    <item>
     <widget class="QGroupBox" name="recorderBox" >
      <property name="title" >
       <string>Recorder</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3" >
       <item>
        <widget class="QLabel" name="labelSelected" >
         <property name="text" >
          <string>Selected channel:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="valueSelected" >
         <property name="text" >
          <string>-</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelTimer" >
         <property name="text" >
          <string>No timer</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout" >
         <item>
          <widget class="QLabel" name="labelBrowse" >
           <property name="text" >
            <string>Directory:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="fileEdit" />
         </item>
         <item>
          <widget class="QToolButton" name="buttonBrowse" >
           <property name="text" >
            <string>...</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="buttonRecord" >
         <property name="text" >
          <string>Record</string>
         </property>
         <property name="icon" >
          <iconset resource="images.qrc" >
           <normaloff>:/icons/images/record.png</normaloff>:/icons/images/record.png</iconset>
         </property>
         <property name="checkable" >
          <bool>true</bool>
         </property>
         <property name="checked" >
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer" >
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="statusBox" >
      <property name="title" >
       <string>Status</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4" >
       <item>
        <layout class="QHBoxLayout" name="currentLayout" >
         <item>
          <widget class="QLabel" name="labelCurrent" >
           <property name="text" >
            <string>Currently recording:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="valueCurrent" >
           <property name="text" >
            <string>-</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="timeLayout" >
         <item>
          <widget class="QLabel" name="labelTime" >
           <property name="text" >
            <string>Time:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="valueTime" >
           <property name="text" >
            <string>0</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="remainingLayout" >
         <item>
          <widget class="QLabel" name="labelRemaining" >
           <property name="text" >
            <string>Time remaining:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="valueRemaining" >
           <property name="text" >
            <string>0</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="labelFile" >
         <property name="text" >
          <string>Recording to file:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="valueFile" >
         <property name="text" >
          <string>-</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2" >
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile" >
    <property name="title" >
     <string>&amp;File</string>
    </property>
    <addaction name="actionOpen" />
    <addaction name="actionClose" />
   </widget>
   <widget class="QMenu" name="menuAbout" >
    <property name="title" >
     <string>&amp;Help</string>
    </property>
    <addaction name="actionHelp" />
    <addaction name="actionAbout" />
   </widget>
   <widget class="QMenu" name="menuActions" >
    <property name="title" >
     <string>Actions</string>
    </property>
    <addaction name="actionTimer" />
    <addaction name="actionRecord" />
   </widget>
   <addaction name="menuFile" />
   <addaction name="menuActions" />
   <addaction name="menuAbout" />
  </widget>
  <widget class="QStatusBar" name="statusbar" />
  <widget class="QDockWidget" name="dockWidget" >
   <property name="sizePolicy" >
    <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize" >
    <size>
     <width>175</width>
     <height>105</height>
    </size>
   </property>
   <property name="floating" >
    <bool>false</bool>
   </property>
   <property name="allowedAreas" >
    <set>Qt::LeftDockWidgetArea|Qt::RightDockWidgetArea</set>
   </property>
   <property name="windowTitle" >
    <string>Channel list</string>
   </property>
   <attribute name="dockWidgetArea" >
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents" >
    <layout class="QVBoxLayout" name="verticalLayout_2" >
     <item>
      <widget class="QTreeWidget" name="playlistWidget" >
       <column>
        <property name="text" >
         <string>Channel</string>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QToolBar" name="toolBar" >
   <property name="windowTitle" >
    <string>Recorder Toolbar</string>
   </property>
   <property name="iconSize" >
    <size>
     <width>22</width>
     <height>22</height>
    </size>
   </property>
   <property name="toolButtonStyle" >
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea" >
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak" >
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen" />
   <addaction name="actionTimer" />
   <addaction name="actionRecord" />
  </widget>
  <action name="actionClose" >
   <property name="icon" >
    <iconset resource="images.qrc" >
     <normaloff>:/icons/images/fileclose.png</normaloff>:/icons/images/fileclose.png</iconset>
   </property>
   <property name="text" >
    <string>Close</string>
   </property>
  </action>
  <action name="actionHelp" >
   <property name="icon" >
    <iconset resource="images.qrc" >
     <normaloff>:/icons/images/help.png</normaloff>:/icons/images/help.png</iconset>
   </property>
   <property name="text" >
    <string>Help</string>
   </property>
  </action>
  <action name="actionAbout" >
   <property name="icon" >
    <iconset resource="images.qrc" >
     <normaloff>:/icons/images/tano.png</normaloff>:/icons/images/tano.png</iconset>
   </property>
   <property name="text" >
    <string>About Tano Player</string>
   </property>
  </action>
  <action name="actionOpen" >
   <property name="icon" >
    <iconset resource="images.qrc" >
     <normaloff>:/icons/images/fileopen.png</normaloff>:/icons/images/fileopen.png</iconset>
   </property>
   <property name="text" >
    <string>Open Channel list</string>
   </property>
  </action>
  <action name="actionRecord" >
   <property name="checkable" >
    <bool>true</bool>
   </property>
   <property name="icon" >
    <iconset resource="images.qrc" >
     <normaloff>:/icons/images/record.png</normaloff>:/icons/images/record.png</iconset>
   </property>
   <property name="text" >
    <string>Record</string>
   </property>
  </action>
  <action name="actionRestore" >
   <property name="icon" >
    <iconset resource="images.qrc" >
     <normaloff>:/icons/images/win.png</normaloff>:/icons/images/win.png</iconset>
   </property>
   <property name="text" >
    <string>Restore</string>
   </property>
  </action>
  <action name="actionTimer" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="icon" >
    <iconset resource="images.qrc" >
     <normaloff>:/icons/images/clock.png</normaloff>:/icons/images/clock.png</iconset>
   </property>
   <property name="text" >
    <string>Timer</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="images.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>actionRecord</sender>
   <signal>toggled(bool)</signal>
   <receiver>buttonRecord</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel" >
     <x>166</x>
     <y>212</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonRecord</sender>
   <signal>toggled(bool)</signal>
   <receiver>actionRecord</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>166</x>
     <y>212</y>
    </hint>
    <hint type="destinationlabel" >
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
